streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -296,13 +300,16 @@ public void putAll(List<KeyValue<K, V>> entries) {
 
     @Override
     public synchronized V delete(K key) {
+        Objects.requireNonNull(key, "key cannot be null");
         V value = get(key);
         put(key, null);
         return value;
     }
 
     @Override
     public synchronized KeyValueIterator<K, V> range(K from, K to) {
+        Objects.requireNonNull(from, "<from> cannot be null");
```

Comment:
I added <> because IMO it is not trivial to recognize from the sentence

`from cannot be null`

that `from` is a variable name. Anyways I removed extra characters.